### PR TITLE
[zh-cn]sync kubeadm-reconfigure.md

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-reconfigure.md

### CODE REVIEW

File: content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-reconfigure.md [hunk 1]
```diff
@@ -324,27 +324,18 @@ The configuration is located under the `data.config.conf` key.
...
 
-获取 Pod 名称：
-
-```shell
-kubectl get po -n kube-system | grep kube-proxy
-```
-
 <!--
-Delete a Pod with:
+Delete a Pods with:
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-reconfigure.md:333]: weird change.

**Review by windsonsea — COMMENTED** [review]
/approve

**Comment by tengqm** [comment]
whoops .... I was trying to close and reopen then see if the merge blocker check can be retriggered.
It appears that the PR immediately after I send `/close`.
So ... is it really merged?

**Comment by my-git9** [comment]
Seems merged

**Pull request closed without merging**